frontend/src/@common/components/EmptyMessage/EmptyMessage.tsx

```diff
@@ -0,0 +1,27 @@
+import Flex from '@/@common/components/Flex/Flex';
+import Text from '@/@common/components/Text/Text';
+import theme from '@/styles/theme';
+
+type EmptyMessageProps = {
+  messages: string[];
+};
+const EmptyMessage = ({ messages }: EmptyMessageProps) => {
```

Comment:
```suggestion
interface EmptyMessageProps = {
  messages: string[];
};

const EmptyMessage = ({ messages }: EmptyMessageProps) => {
```

컨벤션에 맞게 수정이 필요해 보입니다!

추가로 줄바꿈도 부탁드려요~